docs/home/whats-new-2-17.md

```diff
@@ -0,0 +1,159 @@
+
+## Deprecation Notifications
+
+Deprecation notifications allow you to plan for future changes in the Run:ai Platform.
+
+### Feature deprecations
+
+Deprecated features will be available for **two** versions ahead of the notification. For questions, see your Run:ai representative. The following features have been marked for deprecation:
+
+* Jobs&mdash;the *Jobs* feature (submission form and view) has been moved to the category of *Legacy*. To enable them, go to *Tools & Settings*, *General*, open the *Workloads* pane, and then toggle the *Jobs view* and *Job submission* switch to the enabled position.
+* Deployments&mdash;the *Deployments* feature has been removed. It has been replaced by *Inference* workloads. For more information, see [Jobs, Workloads, and Workspaces](#jobs-workloads-and-workspaces) above.
+* Workspaces view&mdash;the *Workspaces* menu has been removed. You can now submit a *Workspace* workload using the *+ New workload* form from the *Workloads* table.
+
+### API support and endpoint deprecations
+
+The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. For more information about Run:ai's API support policy and deprecation process, see [Developer overview](../developer/overview-developer.md#overview-developer-documentation).
+
+The following list of API endpoints have been marked for deprecation:
+
+#### Jobs, events, pods API (replaced by workloads/pods/events)
+
+| Deprecated endpoint | Replacement endpoint |
+| -- | -- |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/jobs | https://app.run.ai/api/v1/workloads |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/jobs/count | https://app.run.ai/api/v1/workloads/count |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/jobs/{jobId}/pods | https://app.run.ai/api/v1/workloads/{workloadId}/pods |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/pods | https://app.run.ai/api/v1/workloads/pods |
+
+### Users, Applications, and Groups API
+
+| Deprecated endpoint | Replacement endpoint |
+| -- | -- |
+| https://app.run.ai/v1/k8s/apps | https://app.run.ai/api/v1/k8s/apps |
+| https://app.run.ai/v1/k8s/users | https://app.run.ai/api/v1/k8s/users |
+| https://app.run.ai/v1/k8s/groups | https://app.run.ai/api/v1/authorization/access-rules (groups should no longer be created, you can only add access rules to them) |
+
+#### Cluster metrics v1 (replaced by v2)
+
+| Deprecated endpoint | Replacement endpoint |
+| -- | -- |
+| https://app.run.ai/v1/k8s/clusters/{clusterUuid}/metrics | https://app.run.ai/api/v2/clusters/{clusterUuid}/metrics |
+
+## Breaking changes
+
+Breaking changes notifications allow you to plan around potential changes that may interfere your current workflow when interfacing with the Run:ai Platform.
+
+### Metrics
+
+Be aware that some names of metrics have been changed and/or have been replaced by the Metrics API. For more information, see [Metrics](../developer/metrics/metrics.md#changed-metrics-and-api-mapping).
```

Comment:
@lavianalon is it correct to say "**replaced** by the Metrics API" or should we say that it was added?
Maybe it is also worth mentioning that we recommend start using the APIs instead of fetching the metrics directly, WDYT?

Reply:
I think we should say "added API as the preferred way to consume" or something of that sort.
It should be clear that the metrics are still available in the old form in 2.17